### ISSUE TITLE
Add a stageConfig option to manually define the stage config file

### ISSUE THREAD

**Issue by ftriolet** [Axway-API-Management-Plus/apim-cli#195]
**User story**

We have several API Manager environnements like : 
```
A
B
C
```
We have several backend environnements like :
```
a1,a2,a3
b1,b2,b3
c
```
we want to expose api like this :
```
A <-> a1,a2,a3
B <-> b1,b2,b3
C <-> c
```
so for doing that we have environnement configuration file :
```
env.A.properties
env.B.properties
env.C.properties
```
and we would like to be able to create api configuration file like that :
```
api-config.a1.json
api-config.a2.json
...
api-config.c.json

```

the option -s (stage) will be used to target the environnement configuration file.
An other option (to create) will be used to target the api configuration file if provided else the value of stage is used

**Additional context**
we are implementing a new CI/CD pipeline and we have many backend environnements and a fixed number of API Manager environnements. We would be able to add backend env without create new API Manager configuration files

Example : 
```
apim api import -s A -c api-conf.json -as a1

with the following api configuration files :
api-conf.json
api-conf.a1.json 
```

**Comment by cwiechmann** [comment]
Hi @ftriolet, If I understood it correctly, that is already supported exactly as you described it. See here:
https://github.com/Axway-API-Management-Plus/apim-cli/wiki/8.1.-Staging-Concept

**Comment by cwiechmann** [comment]
Ah, I see. You need another option to define an override just for the API-Config still pointing to the same API-Manager environment. I'm not sure, how the relation is between API-Manager environments and Backend-Environments, but you could just create multiple environments. 
E.g. 
a1, a2, a3 --> All are still pointing to the same API-Manager, but give you the option to use different overrides for your API-Configuration.

**Comment by ftriolet** [comment]
in our CI/CD, environnement configuration file are in a private repository non easily updatable . These files are injected in a Jenkins slave during the process.

**Comment by blazmanx34** [comment]
To complete the request of @ftriolet.
Api project environnements are differents of API-Manager environments.

The number of API-Manager environments is fixed but the number of api project environnements is variable (and extensible).

Some api projects can have 1 or 2 or 3... development environments and all these api projects must be expose through the developement API-Manager environment. Idem for staging, qa... project environments.

We know how to match an api project environment with an API-Manager environment but apim-cli requires that both have the same name (and it's impossible in our case).
We also can't afford to add a new environment.configuration file  (or symlink) every time a projet create a new environment (for exemple a fourth developement environnement)

**Comment by cwiechmann** [comment]
Thanks for the additional information @blazmanx34. Technically, it should be very simple to implement, so let's do this as is creates more flexibility.

**Comment by cwiechmann** [comment]
@ftriolet, @blazmanx34, the new parameter `-stageConfig` has been added and will be released with version 1.3.11. 

This is the belonging documentation:
https://github.com/Axway-API-Management-Plus/apim-cli/wiki/8.1.-Staging-Concept#stage-config

Let me know, if you have any questions or comments. I think this make the issue #202 completely obsolete and I'll close this now.
